src/configs/javascript.ts

```diff
@@ -117,7 +117,12 @@ export const javascript = {
             },
         ],
         'no-unused-expressions': 'error',
-        'no-unused-vars': 'error',
+        'no-unused-vars': [
+            'error',
+            {
+                argsIgnorePattern: '^_',
```

Comment:
We should ignore the scenarios that affect us instead of allowing any var to be prefixed with `_`
```suggestion
                args: 'after-used',
                ignoreRestSiblings: true,
```